conf/configuration.go

```diff
@@ -69,7 +69,7 @@ type ExternalProviderConfiguration struct {
 type SMTPConfiguration struct {
 	MaxFrequency time.Duration `json:"max_frequency" split_words:"true"`
 	Host         string        `json:"host"`
-	Port         int           `json:"port"`
+	Port         int           `json:"port" envconfig:"PORT" default:"587"`
```

Comment:
This envconfig tag will conflict with the API port. https://github.com/netlify/gotrue/blob/a4579e3641fb9f1c443de50a2bd77c53a101d02f/conf/configuration.go#L40-L44